my.css

```diff
@@ -1,97 +1,376 @@
-body {
-  background-color: #b6bddf;
-  font-weight: 600;
-  font-family: 'Times New Roman', Times, serif;
-  margin: 0;
+*,*::before,*::after{
   padding: 0;
+  margin: 0;
+  border: 0;
+  box-sizing: border-box;
 }
-.qqqq {
-  background-color: white;
+
+html, body{
   height: 100%;
-  overflow: auto;
-  width: 500px;
+  font-family: Arial, Helvetica, sans-serif;
 }
-.qqqq a {
-  display: block;
-  text-decoration: none;
-  color: black;
-  padding: 16px;
+
+.container {
+  max-width: 1200px;
+  margin: 0px auto;
+  padding: 0px 25px;
 }
-.qqqq a.wwwww {
-  background-color: green;
-  color: white;
+.header {
+  position: fixed;
+  width: 100%;
+  top: 0;
+  left: 0;
+  z-index: 50;
 }
-.qqqq a:hover {
-  background-color: grey;
-  color: white;
+.header::before{
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgb(97, 97, 204);
+  z-index: 2;
+
 }
-.qqqq a.active {
-  background: red;
-  color: white;
+.headerbody {

```

Comment:
Please choose some code style for classnames like a camel case or using _ or - between words.

Reply:
ok